mode/examples/Topics/Image Processing/Blending/Blending.pyde

```diff
@@ -21,17 +18,17 @@ modes = ((REPLACE, "REPLACE"),
          (MULTIPLY, "MULTIPLY"),
          (SCREEN, "SCREEN"),
          (REPLACE, "REPLACE"))
+img1 = loadImage("layer1.jpg")
```

Comment:
Well. That stinks. The root cause was obviously something else. I just tried a minimal example, and it's fine.